docs/jetstream/configuration.md

```diff
@@ -326,3 +328,73 @@ data_source = "search_clients_engines_sources_daily"
 [metrics.urlbar_amazon_search_count.statistics.bootstrap_mean]
 [metrics.urlbar_amazon_search_count.statistics.deciles]
 ```
+
+### Defining Exposure Signals
+
+Many Nimbus features will send a [Nimbus exposure event] automatically when the feature configuration is consulted;
+these are `normandy#expose` events on desktop and `nimbus_events.exposure` events in Glean.
+However, it is also possible to define custom exposure events:
+
+```toml
+[experiment.exposure_signal]
+name = "nimbus"
+friendly_name = "Nimbus exposure signal"
+description = "Nimbus desktop feature exposure signal"
+data_source = "events_unnested"
+select_expression = "
+   event_category = 'normandy'
+   AND event_method = 'expose'
+   AND event_object = 'nimbus_experiment'
+   AND event_string_value = 'experiment-slug'
+   AND normalized_channel = 'release'
+"
+window_start = 0                    # optional
+window_end = "analysis_window_end"  # optional
+
+
+[data_sources.events_unnested]
+from_expression = """(
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.telemetry.events`
+)"""
+```
+
+`select_expression` defines the condition for when an exposure happens. Exposure signals can use predefined or custom `data_sources`. `window_start` and `window_end` are optional parameters that specify the date range when clients are checked for exposure.
+The parameters are representing the number of days before or after the first enrollment date. There are some additional custom values that can be used:
+* `enrollment_start`: equal to using `0`
+* `enrollment_end`: equal to the enrollment period length in days
+* `analysis_window_start`: to use the start of the current analysis window
+* `analysis_window_end`: to use the end of the current analysis window
+* some integer: to specify the window with a custom value
+* if parameters are not specified: default to `enrollment_start` and `enrollment_end`
+
+Metrics based on clients that have seen the exposure signal are only computed for those that specify `enrollments` as one of their `analysis_bases`:
```

Comment:
Just double checking that this is correct. Should it say "only computed for those that specify `enrollments`" or "only computed for those that specify `exposures`"?

Reply:
whoops, good catch. It should be `exposures`.